orbit/hyperspace.py

```diff
@@ -75,35 +69,34 @@ def do_create_session(args):
 
 def do_validate_creds(args):
     need(args, u=True, p=True)
-    u, p = args.username, args.password
-    pwdhash = db.usr_pwdhashfor_username(u)[0]
-    if pwdhash is None:
-        nou(u)
-    pwdhash = pwdhash[0]
-
-    if bcrypt.checkpw(bytes(p, "UTF-8"), bytes(pwdhash, "UTF-8")):
-        print('credentials(username: {}, password:{})'.format(u, p))
-    else:
+    if not (user := db.User.get_or_none(db.User.username == args.username)):
+        nou(args.username)
+    if not bcrypt.checkpw(args.password.encode('utf-8'),
```

Comment:
this is basically check_credentials, in future could be helpful to use the other code to reduce repetition

Reply:
honestly wondering why hyperspace can even do this tbh. I think I am going to just axe it in a follow up pr. Just logging into the website is probably easier than calling this tool.